packages/utilities/package.json

```diff
@@ -40,4 +42,4 @@
     "react": "^0.14 || ^15.0.1-0 || ^16.0.0-0",
```

Comment:
Do we have to upgrade to 0.14.9 now? 

https://github.com/facebook/prop-types#compatibility